### ISSUE TITLE
Where are the loss function changes in the codebase?

### ISSUE THREAD

**Issue by plamb-viso** [r-three/t-few#25]
From the paper: "As an objective, we use the sum of a standard language modeling loss, an unlikelihood loss for incorrect choices, and a length-normalized loss."

And the code also uses huggingface transformers. I was just wondering if you could point me to where the loss function is modified and then used in training in the codebase.

**Comment by HaokunLiu** [comment]
src/model/EncoderDecoder.py , training_step function. In our experiments, we have self.config.mc_loss = 1, self.config.unlikely_loss = 1, self.config.length_norm = 1, you can follow the if clauses there to find the implementations.
